templatetags/htk_tags.py

```diff
@@ -272,6 +273,20 @@ def is_editable_by_context_user(context, obj):
         is_editable = False
     return is_editable
 
+@register.simple_tag(takes_context=True)
+def has_permission(context, permission_codename):
```

Comment:
what if you just pass in the full `permission_key` instead of `permission_codename`? saves the additional lookup and stringbuilding

Reply:
This would mean the user needs to know the app_label as well. I think it will be easier if the user didn't have to worry about that.

Reply:
I made up the term, I tried looking for something. Would you be okay if we had two params in this case to be extra clear. `app_label` and `permission_codename` or should we just go with `permission_key`?

Reply:
the latter. one.

Django calls it `perm`, which is rather inane

Reply:
but feel free to call it `perm` as well, if that makes you feel better